packages/node_modules/@webex/plugin-logger/README.md

```diff
@@ -24,13 +25,11 @@ This is a plugin for the Cisco Webex JS SDK . Please see our [developer portal](
 
 ```js
 const Webex = require('webex');
+require('@webex/plugin-logger');
```

Comment:
Plugin logger is [already part of `webex`](https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/webex/src/webex.js#L20) why does it need to be `require`'d again?